t/082-body-filter-2.t

```diff
@@ -34,7 +34,7 @@ log_level('debug');
 
 repeat_each(2);
 
-plan tests => repeat_each() * (blocks() * 5);
+plan tests => repeat_each() * (blocks() * 5 - 4);
```

Comment:
Better to use a positive number, so the plan number won't be negative when `--- ONLY` is given.

Reply:
Done